var/spack/repos/builtin/packages/nmodl/package.py

```diff
@@ -14,7 +14,8 @@ class Nmodl(CMakePackage):
     git      = "https://github.com/BlueBrain/nmodl.git"
 
     version('develop', branch='master', submodules=True)
-    version('0.3', commit="86fc52d2", submodules=True)
+    version('0.4a', commit="c30ea06", submodules=True)
+    version('0.3a', commit="86fc52d", submodules=True)
```

Comment:
Are we just skipping some versions?

Reply:
@matz-e : you are right! Actually, we haven't released 0.3 yet and it will be tagged soon (so it was a mistake). As we haven't used/deployed nmodl package on BB5, I am turning above versions to 0.3a and 0.3b.